examples/demo.css

```diff
@@ -249,4 +249,10 @@ h1 {
 a.name-header{
   text-decoration: none;
   color: #445;
+}
+.panel.panel-default{
+  position: relative;
+  top: 0;
+  right: 0;
+  width:1100px;
```

Comment:
Ah hmm, not sure we want to set a fixed width - try it at different screen widths! Currently it resizes naturally.

Reply:
Yeah it's resizing normally. Removed the line.